packages/webapp-libs/webapp-finances/src/components/stripe/stripePaymentForm/__tests__/stripePaymentForm.component.spec.tsx

```diff
@@ -56,40 +58,71 @@ describe('StripePaymentForm: Component', () => {
   };
 
   const Component = (props: Partial<StripePaymentFormProps>) => (
-    <Elements stripe={null}>
-      <StripePaymentForm {...defaultProps} {...props} />
-    </Elements>
+    <CurrentTenantProvider>
+      <Elements stripe={null}>
+        <StripePaymentForm {...defaultProps} {...props} />
+      </Elements>
+    </CurrentTenantProvider>
   );
 
   const selectProduct = async (value = TestProduct.A) =>
     userEvent.click(await screen.findByRole('radio', { name: `$${value}` }));
   const sendForm = async () => userEvent.click(await screen.findByRole('button', { name: /Pay \d+ USD/i }));
 
   it('should render without errors', async () => {
-    const requestMock = fillAllPaymentsMethodsQuery(allPaymentsMock as Partial<Subscription>[]);
+    const tenantId = 'tenantId';
```

Comment:
You can move this var to the upper scope and initialize it once instead of multiple times in each test

Reply:
sure, I initially thought that it's better to keep all the mocks in the context of each test but actually here we don't get much value out of it.